test/receiverSpec.js

```diff
@@ -529,18 +529,43 @@ describe('Receiver Module', function () {
           const kep = this.receiver.generateSharedSecret()
           const request = sender.createRequest(Object.assign({}, kep, {
             destination_amount: "1",
+            data: { for: 'that thing' }
           }))
 
           this.transfer.account = request.address
           this.transfer.executionCondition = request.condition
           this.transfer.data.ilp_header.data.expires_at = request.expires_at
           this.transfer.data.ilp_header.account = request.address
-          delete this.transfer.data.ilp_header.data.data
+          this.transfer.data.ilp_header.data.data = request.data
```

Comment:
Can you add a test specifically for the fact that the sender automatically encrypts the data, and that the receiver expects it to be encrypted? Also, what would happen if the data is not encrypted? Would the receiver understand it?